test/features/navigation.feature

```diff
@@ -0,0 +1,23 @@
+@smoke
+Feature: Navigation
+
+    Scenario: DQL-65 BDD-1 "Request data" link in header (non-logged in user)
```

Comment:
please remove `DQL-*` prefix

Reply:
Jira story references removed from Scenario descriptions and Scenarios elaborated as per William's request on other BDD PRs.